pyat/at/lattice/elements.py

```diff
@@ -1156,11 +1158,63 @@ def __init__(self, family_name: str, betax: float = 1.0,
         if espread > 0.0:
             assert tauz > 0.0, 'if espread is given, tauz must be non zero'
 
-        self.U0 = U0
         self.betax = betax
         self.betay = betay
         super(SimpleQuantDiff, self).__init__(family_name, **kwargs)
 
+class SimpleRadiation(_DictLongtMotion, Radiative, Element):
+    """Simple radiation damping and energy loss"""
+    _BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES
```

Comment:
With this, `repr(simple_radiation_element)` will not work. To make it work, you would need
```python
_BUILD_ATTRIBUTES = Element._BUILD_ATTRIBUTES + ['taux', 'tauy', 'tauz']
```
And have `taux`, `tauy` and `tauz` properties of the element (deduced from dampx,...).

This is not critical, we can live without it… Up to you.

Reply:
I added the taux,y,z to build attributes but

>  And have taux, tauy and tauz properties of the element (deduced from dampx,...).

this requires some additional action?

Reply:
It implies a significant refactoring of the constructor. Example: tests on argument validity are done only at element creation. It is better to do them in `_conversions` functions, so that they are also done if someone modifies an attribute. Here, `taux` would be a property, with a `_conversions` function requiring it to be positive. A user could modify `taux`, and it would be checked the same way.

For now, you can forget it, we can do that later.